pyproject.toml

```diff
@@ -1,6 +1,6 @@
 [project]
 name = "craft-archives"
-version = "0.0.3"
+version = "0.0.4"
```

Comment:
we should start a changelog.rst as we have for the others. Next version bump should have (and potentially backfilled)